docs/components/index.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Blazor components
+author: guardrex
+description: Learn how to create and use Blazor components, the fundamental building blocks of Blazor apps provided by compiled Razor or C# files.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 04/09/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/components/index
+---
+# Blazor components
+
+By [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/client-side/blazor/components/common/samples/)
+
+To download the sample:
+
+1. Download and unzip the [Blazor.Docs repository zip file](https://codeload.github.com/aspnet/Blazor.Docs/zip/master).
+1. Use the URL in the [sample link](https://github.com/aspnet/Docs/tree/master/aspnetcore/client-side/blazor/components/common/samples/) to navigate to the sample folder.
```

Comment:
I might be being dim, but I don't understand this second list item at all. What URL? What sample folder? And how do you *run* the samples after you've downloaded them?

Reply:
One thing that *LEAPS* right out is the link itself, which changes now that this is in the Blazor.Docs repo (that link was held over from when this PR sat in the aspnet/Docs repo). I'll fix it on this pass.

Anyway ... to your point ... This topic has a sample to accompany it. The dev can click this link (after I fix it) and get to the sample app. They take the ZIP of the docs repo to download the sample. Don't ask!! 🙈 lol ... We've had a lot of discussion about that, and we have a fix on the way to make sample downloading a snap. They run the sample the same way they would run any standalone Blazor app. [We probably need to address that further tho. Let's discuss this further on the second pass after the next commit.]

The sample matches most of what the topic covers. It also (the way I build them) is completely self-documenting. If a dev ONLY has the sample, they can still understand how the Blazor app works. When you run it, it shows the Razor code behind the component that makes the component work ... it shows that code in the sample's rendered output. This has been a very effective way to build samples when training devs on ASP.NET Core features (but I admit that my conclusion is based on my anecdotal experience talking to devs).

Now that we aren't going have the "Build your first" tutorial in the launch doc set, I think this sample app is even *MORE* important. There seems to be very few (or no) sample apps explained in detail (outside of your presentations/videos).